dotnet/src/Experimental/Assistants/IChatMessage.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.ObjectModel;
+
+namespace Microsoft.SemanticKernel.Experimental.Assistants;
+
+/// <summary>
+/// Represents a message that is part of an assistant thread.
+/// </summary>
+public interface IChatMessage
```

Comment:
We already have this abstraction in core, can we extend that?

Reply:
I'm not sure the binding has more value than headache.  Can we take that as a follow-up please?  (Making a list).